popin-action-page.php

```diff
@@ -0,0 +1,19 @@
+<?php get_header(); ?>
+
+	<div class='main-section'>
+	  <ul class="posts">
+		<?php 
+		  if ( have_posts() ) :
+	      	while ( have_posts() ) : the_post();
```

Comment:
When I learnt about this it seemed like this page will post form details which will never be visible but behave as other accessible pages. I know you have a better way, shoot!